### PR TITLE
Import definitions from usfs

### PR DESCRIPTION
**Pull request by annekainicUSDS**
We're moving some of the definitions that are pretty specific and potentially not generally usable back into vets-website. This is the first pass at doing that.

Accompanying PR on USFS: https://github.com/usds/us-forms-system/pull/214

### CODE REVIEW

**Review by cehsu — APPROVED** [review]
LGTM, some references in `src/platform/forms/pages/applicantInformation.jsx` should be updated as well.

**Review by rianfowler — APPROVED** [review]
Looks good. I was able to go t through the form 1900 locally without any issues and I only found one thing that wasn't swapped out (Claire noted it already).

File: src/applications/burials/config/form.js [hunk 1]
```diff
@@ -23,10 +23,10 @@ import { isFullDate } from '../../../platform/forms/validations';
 import { externalServices } from '../../../platform/monitoring/DowntimeNotification';
 import GetFormHelp from '../../../platform/forms/components/GetPensionOrBurialFormHelp';
 import FormFooter from '../../../platform/forms/components/FormFooter';
+import fullNameUI from '../../../platform/forms/definitions/fullName';
 import environment from '../../../platform/utilities/environment';
 
-import * as address from 'us-forms-system/lib/js/definitions/address';
-import fullNameUI from 'us-forms-system/lib/js/definitions/fullName';
```
[the remark this answers is not in the document]
Comment (annekainicUSDS) [reply src/applications/burials/config/form.js]: Ah thanks, I missed those!

**Comment by annekainicUSDS** [comment]
@cehsu @rianfowler for some reason I'm now getting an issue where it's saying that the `environment` variable isn't defined in `src/applications/edu-benefits/complaint-tool/helpers.js`. I'm stumped on how my changes might have caused this issue. Do either of you have an idea?

**Comment by annekainicUSDS** [comment]
Looks like that lint error is happening on master as well, so perhaps it was introduced by a different PR?

**Comment by rianfowler** [comment]
@annekainicUSDS We had a merge collision that was resolved improperly. I've just merged the [fix](https://github.com/department-of-veterans-affairs/vets-website/pull/8152)- you should now be able to pull master and successfully build.

**Pull request merged**